core/templates/dev/head/pages/exploration_player/learner_local_nav.html

```diff
@@ -7,24 +7,19 @@
     </a>
   </li>
   <li>
-    {% if can_edit %}
-      <a ng-href="/create/<[explorationId]>" uib-tooltip="<['I18N_PLAYER_EDIT_TOOLTIP' | translate]>" tooltip-placement="bottom" target="_blank">
+      <a ng-if="canEdit" ng-href="/create/<[explorationId]>" uib-tooltip="<['I18N_PLAYER_EDIT_TOOLTIP' | translate]>" tooltip-placement="bottom" target="_blank">
```

Comment:
Extra 2 spaces at the beginning of this line. (Also indent inner tags and closing tags)